scout/load/report.py

```diff
@@ -1,159 +1,26 @@
 # -*- coding: utf-8 -*-
 import logging
 
-from scout.adapter import MongoAdapter
+from scout.constants import CUSTOM_CASE_REPORTS
 from scout.exceptions import DataNotFoundError, IntegrityError
+from scout.server.extensions import store
 
 LOG = logging.getLogger(__name__)
 
 
-def load_delivery_report(
```

Comment:
These methods are a lot repetitive and can be replaced by a few lines of code instead, see at the bottom of this file